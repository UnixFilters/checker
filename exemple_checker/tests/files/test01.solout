{
    "steps": [
        {
            "number": 0,
            "command_string": "cat",
            "output": "",
            "stderr": "",
            "return": 0
        }
    ]
}
