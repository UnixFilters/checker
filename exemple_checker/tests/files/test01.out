1997
1965
1960
1954
1952